dreambooth/optimization.py

```diff
@@ -573,65 +572,65 @@ def log_dadapt(disable: bool = True):
     else:
         return 5
 
-def get_optimizer(args, params_to_optimize):
+
+def get_optimizer(optimizer: str, learning_rate: float, weight_decay: float, params_to_optimize):
```

Comment:
Now that we have separate fields for Lora LR rates, we need to pass the learning rate rather than passing args and using args.learn_rate in this function.